src/api/app/assets/stylesheets/webui/label.scss

```diff
@@ -0,0 +1,4 @@
+.label {
+  padding: 0.25rem 0.5rem;
+  border-radius: var(--bs-border-radius);
```

Comment:
In my tests I've been using `d-inline-block` on the elements using this class to get a bit of spacing between the labels. I guess we should add the equivalent here.

Reply:
Done